analysis/condor/runEfficienciesQED.sh

```diff
@@ -1,32 +1,32 @@
 #!/bin/bash
 
-configPath="/afs/cern.ch/work/j/jniedzie/private/LightByLight2018/analysis/configs/efficiencies.md"
```

Comment:
How about you create a variable storing base path, such that we don't have to change it everywhere in the script?

Reply:
Incorporated all changes requested.